docs/faststore/components/features/auth-content.mdx

```diff
@@ -0,0 +1,185 @@
+
+| Before log in | After log in |
+| ------------- | ------------ |
+| ![auth-content-before-log-in](https://vtexhelp.vtexassets.com/assets/docs/src/before-login___1c428d5e4b755d2c94fb64968893290c.png) | ![auth-content-after-log-in](https://vtexhelp.vtexassets.com/assets/docs/src/after-login___de2dbeae277be2cae93ecc9ec51b3929.png) |
+
+Authenticated content fosters a more controlled and secure shopping environment for stores and customers, enhancing the shopping experience. For example, in B2B settings, this allows for customized access to product catalogs, ensuring that businesses only see relevant products and pricing. For segmented promotions, authenticated content enables merchants to tailor special offers and discounts to specific customer segments, increasing engagement and conversion rates.
+
+In this guide, learn how to create an authenticated component using the [Hero](https://developers.vtex.com/docs/guides/faststore/organisms-hero) component as an example.
+
+## Before you begin
+
+### Update the `@faststore/core` package
+
+Ensure your store is using `@faststore/core@v3.0.39` or later. To benefit from this feature, follow the steps below:
+
+1. Update your store’s `@faststore/core` package to the `v3.0.39` by running the following command:
+
+    ```bash
+    yarn add @faststore/core@3.0.39
+    ```
+
+2. Run `yarn dev` to apply the changes to your project.
+
+### Understand Section Override
+
+Make sure you understand the concepts of [Section Override](https://developers.vtex.com/docs/guides/faststore/overrides-overview) to follow along.
+
+## Step by step
+
+### Step 1 - Override a component
+
+Choose the component you want to restrict access to. To check the available components, refer to the [UI components](https://developers.vtex.com/docs/guides/faststore/components-index) guide.
+
+Once you have chosen the component, you will need to override the component using the feature [Section Override](https://developers.vtex.com/docs/guides/faststore/overrides-overview). In this guide we will use the [Hero](https://developers.vtex.com/docs/guides/faststore/organisms-hero) component as an example.
+
+1. In `src/components/overrides` create the `Hero.tsx` file.
+2. Add the following to the `Hero.tsx` file:
+
+    ```tsx
+    import { SectionOverride } from "@faststore/core";
+    import { HeroHeader, HeroImage } from "@faststore/ui";
+
+    const SECTION = "Hero" as const;
+
+    const override: SectionOverride = {
+    section: SECTION,
+    components: {
+        HeroImage: {
+        Component: (props) => (
+            <HeroImage {...props} />
+        ),
+        },
+    },
+    };
+
+    export { override };
+
+    ```
```

Comment:
This code block and others in this documentation are not rendering properly on [markdown-preview](https://developers.vtex.com/editor/markdown-preview) -- for me they appear as inline code. Not sure if the issue is with the preview, because I couldn't find a reason in the code itself, even testing alternatives. I recommend keeping an eye on this when publishing.

Reply:
Not sure either 🤔
The [doc preview](https://developers.vtex.com/docs/guides/faststore/features-auth-content) seems alright
<img width="529" alt="Screenshot_524" src="https://github.com/vtexdocs/dev-portal-content/assets/67270558/c3067da6-f31e-4bf9-90e8-e445fe046229">